src/instance.rs

```diff
@@ -463,6 +463,13 @@ where
 }
 
 impl<T> Py<T> {
+    /// Returns whether the object is the same as the other.
+    ///
+    /// This is equivalent to the Python expression `self is other`.
+    pub fn is<U: AsPyPointer>(&self, o: &U) -> bool {
```

Comment:
Should this inherit the `#[inline]` attribute from the `PartialEq` method?

Reply:
I think probably makes sense for this to be `#[inline]`.

Reply:
will do